pkg/ntpmonitor/config.go

```diff
@@ -37,14 +43,20 @@ type Config struct {
 // Config object with the vales found. No sanity checking is made of the
 // loaded values.
 func LoadConfig(path string) (*Config, error) {
-	data, err := os.ReadFile(path)
-	if err != nil {
-		return nil, fmt.Errorf("failed to read file: %s %w",
-			path, err)
+	var configData []byte
+	if path == "ntpsync.yaml" {
```

Comment:
```suggestion
	if path == "" {
```

Reply:
Due to the change of the default flag

Reply:
Just tested that the server works locally with this change now.